news_scraper/spiders/businessstandard.py

```diff
@@ -16,7 +16,7 @@ class BusinessStandardSpider(SitemapIndexSpider):
         "month": lambda d: d.strftime("%B").lower(),
     }
 
-    sitemap_rules = [(r"/markets/", "parse")]
+    sitemap_rules = [(r"/markets/", "parse"),(r"article/markets/", "parse"),(r"article/companies/", "parse"),(r"article/economy-policy/", "parse")]
```

Comment:
@ARAVINT-S-A  Are you sure it's `article/markets/` and not just `/markets/` along with the others? These rules need to be updated

Reply:
https://www.business-standard.com/article/markets/sebi-slaps-rs-51-lakh-penalties-on-18-entities-in-atl-case-122033101539_1.html

I saw few pages like this so I included that

Reply:
`/markets/` and `article/markets` are redundant. Unless something like `/something-else/markets` kind of urls also exist, this doesnt make sense. 

Either remove `/markets/` and keep all other categories with `/article/` like `/article/news`or drop `/article` and keep everything like `/news/`.

The trailing slash is important `/article/markets/` is not the same as `article/markets`, a bit paedantic but better to be safe.